python_modules/dagster/dagster/_core/definitions/decorators/schedule_decorator.py

```diff
@@ -41,6 +52,15 @@ def schedule(
     job: Optional[ExecutableDefinition] = None,
     default_status: DefaultScheduleStatus = DefaultScheduleStatus.STOPPED,
     required_resource_keys: Optional[Set[str]] = None,
+    target: Optional[
+        Union[
+            "CoercibleToAssetSelection",
+            "AssetsDefinition",
+            "JobDefinition",
+            "UnresolvedAssetJobDefinition",
+            "GraphDefinition",
```

Comment:
should we still be taking a `GraphSelection` here?

Reply:
Could go either way but I think so. `GraphDefinition` is still accepted by `job` argument (deprecated for removal in 2.0). Keeps things simpler internally and from a UX perspective to just have `target` be a superset of job rather than also excluding this rarely used case that will be removed later anyway.

Reply:
I think we should just remove it since it is a greenfield argument.